lib/shinseibank/cli/account.rb

```diff
@@ -0,0 +1,36 @@
+require "ostruct"
+require "shinseibank/cli/subcommand"
+
+class ShinseiBank
+  class CLI < Thor
+    class Account < Subcommand
+      desc "show", "Show your account details"
+      def show
+        login
```

Comment:
feels like this is going to be required for all entrypoints. (logout would be nice too)
Is there any way to wrap all calls between those 2?